src/www/ui/api/index.php

```diff
@@ -111,6 +111,7 @@ function (){
     $this->post('', UploadController::class . ':postUpload');
     $this->get('/{id:\\d+}/summary', UploadController::class . ':getUploadSummary');
     $this->get('/{id:\\d+}/licenses', UploadController::class . ':getUploadLicenses');
+    $this->options('', AuthController::class . ':optionsVerification');
```

Comment:
Better to use path `'[/{params:.*}]'` instead of empty path in all options catch. This might fail a preflight call for `DELETE /api/v1/uploads/8`.

Reply:
okay @GMishx